root/io/filemerger/CMakeLists.txt

```diff
@@ -340,7 +340,7 @@ if(${compression_default} STREQUAL "zlib")
          if(CMAKE_SYSTEM_PROCESSOR MATCHES aarch64)
             ROOTTEST_ADD_TEST(simple-lz4-compr-level9
                           PRECMD ${ROOT_hadd_CMD} -f409 hsimple409.root hsimple.root
-                          COMMAND ${ROOT_root_CMD} -q -l -b "${CMAKE_CURRENT_SOURCE_DIR}/testSimpleFile.C(\"hsimple409.root\",25000,409,516975,5)"
+                          COMMAND ${ROOT_root_CMD} -q -l -b "${CMAKE_CURRENT_SOURCE_DIR}/testSimpleFile.C(\"hsimple409.root\",25000,409,516975,10)"
```

Comment:
This is already in the `aarch64` sections and thus the difference is from prior/different runs on arm64.  Do we understand what changed?

Reply:
I have no idea what changed. It's the first run of the new alma9-arm runners.

`git blame` says:
```
Author: Oksana Shadura <oksana.shadura@cern.ch>
Date:   Mon Jul 8 11:47:27 2019 +0200

    Adjust file size for LZ4 (409 for aarch64)
```
Maybe the change is 5 years of evolution of distro and lz4. I'm not sure if this test ran in the mean time.

Reply:
Do you think it's safe to update the size? 7 out of 517k bytes seems like a small change.

Reply:
I updated to the commit message to be a bit clearer.

Reply:
Thanks.  We should also clarify if it is used the system lz4 or the builtin version.